crates/re_types_builder/src/codegen/python.rs

```diff
@@ -590,7 +592,9 @@ fn code_for_union(
 
     code.push_text(quote_doc_from_docs(docs), 0, 4);
 
-    if !init_func.is_empty() {
+    if init_func.is_empty() {
+        code.push_text("# You can define your own __init__ function by defining a function called {init_override_name:?}", 2, 4);
```

Comment:
This isn't formatted! As in this will literally output `{init_override_name:?}`

Reply:
haha, lol. We should have a lint about this :)